server/src/graphql/queries/envVars.ts

```diff
@@ -0,0 +1,22 @@
+import { appNameSchema } from './../utils';
+import { QueryResolvers } from '../../generated/graphql';
+import { sshConnect } from '../../lib/ssh';
+import { dokku } from '../../lib/dokku';
+
+export const envVars: QueryResolvers['envVars'] = async (
+  _,
+  { name },
+  { userId }
+) => {
+  if (!userId) {
+    throw new Error('Unauthorized');
+  }
+
+  appNameSchema.validateSync({ name });
+
+  const ssh = await sshConnect();
```

Comment:
before this part for the query and both mutations we should do a findOne to check that the app exist in the db first, if does not exist, throw an error